zhaquirks/nodon/pilot_wire.py

```diff
@@ -0,0 +1,100 @@
+"""NodOn pilot wire heating module."""
+
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import EntityType, QuirkBuilder
+import zigpy.types as t
+from zigpy.zcl.foundation import (
+    BaseAttributeDefs,
+    BaseCommandDefs,
+    DataTypeId,
+    Direction,
+    ZCLAttributeDef,
+    ZCLCommandDef,
+)
+
+NODON = "NodOn"
+NODON_MANUFACTURER_ID = 4747
+NODON_PILOT_WIRE_CLUSTER_ID = 0xFC00  # 64512
+ADEO = "Adeo"
+
+
+class NodOnPilotWireMode(t.enum8):
+    """Pilot wire mode."""
+
+    # Codes taken from
+    # https://github.com/Koenkk/zigbee-herdsman-converters/blob/0f4833340a20db3dae625a61c41d9be0a6f952be/src/converters/fromZigbee.ts#L5285.
+
+    Off = 0x00
+    Comfort = 0x01
+    Eco = 0x02
+    FrostProtection = 0x03
+    ComfortMinus1 = 0x04
+    ComfortMinus2 = 0x05
```

Comment:
I just remembered that https://github.com/zigpy/zha-device-handlers/pull/3031 has the same enum.
In that PR, we wanted to implement it as additional modes for the climate entity (so partly in ZHA).

I'll have to think about what we want to do here now.

Reply:
Tbh, we can probably just add it like this for now (select entity for the enum)..
If we ever get to implementing it as a climate entity in HA, there might be a breaking change though, as this would eventually be removed (and/or disabled by default).

But I guess having this would be better than not having it at all for a while.